File: 백준/Bronze/10818. 최소， 최대/최소， 최대.cc
#include <iostream>

using namespace std;

int main(void) {
    int N, max = -1000000, min = 1000000;
    
    cin >> N;
    int num[N];
    
    for (int i = 0; i < N; i++) {
        cin >> num[i];    
        if (max < num[i]) {
            max = num[i];
        }
        if (min > num[i]) {
            min = num[i];
        }
    }
    cout << min << " " << max << endl;
    return 0;
}

File: Java/백준/Bronze/10818. 최소， 최대/최소， 최대.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int n = s.nextInt();
        int min = 1000000, max = -1000000;
        int num;
        
        for (int i = 0; i < n; i++) {
            num = s.nextInt();
            if (num > max) max = num;
            if (num < min) min = num;
        }
        System.out.printf("%d %d", min, max);
        s.close();
    }
}